### ISSUE TITLE
Bug: Authentication not working with USESSL=false

### ISSUE THREAD

**Issue by Vincent-Nikkelen-mkb** [allmannyttan/matcha-kontraktet-deployment#6]
When configuring USESSL=false, the users browser tries to make a call to
```http://localhost/backend/auth/generate-token```

As this call is made from the users browser, there is no process on the users system to answer this call.

**Comment by anderslarsson** [comment]
I guess the problem here is that the comment implies you are running Matcha locally without ssl. But you could test/run this in a cloud environment with a domain without ssl. So the comment should be changed.

**Comment by Vincent-Nikkelen-mkb** [comment]
Yes, I can confirm we were trying to run this in a Cloud environment.

I would propose to rename USESSL as a parameter to RUNLOCALLY. That would be much more descriptive.
